### ISSUE TITLE
Accessibility improvements - color contrast, translations and tooltips

### ISSUE THREAD

**Issue by robinmolen** [open-formulieren/open-forms#5051]
This ticket describes some small accessibility improvements concerning color contrasts, missing translations and the accessibility of tooltips.

- [ ] 1.4.3 (color contrast) Error messages have a contrast of 3,7:1. This should be 4,5:1
- [ ] 1.4.11 (non-textual color contrast) Checkbox and radio button borders have a contrast of 2,9:1. This should be 3,0:1
- [ ] 2.1.1 (navigation - keyboard) Tooltip cannot be accessed by screen readers
- [ ] 3.1.2 (language) Some texts are non translatable. The rapport mentions the title of the select component delete button.
- [ ] 3.3.3 (error suggestion) When a specific input pattern is expected, the error message should provide the expected format or an example.
